ssz/constants.py

```diff
@@ -1 +1,5 @@
-SSZ_CHUNK_SIZE = 128
+CHUNK_SIZE = 32
```

Comment:
Should it be renamed to `BYTES_PER_CHUNK` as the spec?

Reply:
I tried to name everything that is a number of bytes "size" and everything that is a number of elements "length" as was suggested by Piper in the mega-PR. The spec does (or at least used to) not distinguish between the two meanings cleanly which I found confusing.

`BYTES_PER_CHUNK` would be fine on its own, but to be consistent with the rest of the code, I'd rather keep `CHUNK_SIZE`.

Reply:
Added the spec constant names as comments